src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -886,20 +886,20 @@ subroutine Restart(this, bounds, ncid, flag)
     if(use_luna) then
       call restartvar(ncid=ncid, flag=flag, varname='vcmx25_z', xtype=ncd_double,  &
          dim1name='pft', dim2name='levcan', switchdim=.true., &
-         long_name='Maximum carboxylation rate at 25 celcius for canopy layers', units='umol CO2/m**2/s', &
+         long_name='Maximum carboxylation rate at 25 Celcius for canopy layers', units='umol CO2/m**2/s', &
          interpinic_flag='interp', readvar=readvar, data=this%vcmx25_z_patch)
       call restartvar(ncid=ncid, flag=flag, varname='jmx25_z', xtype=ncd_double,  &
          dim1name='pft', dim2name='levcan', switchdim=.true., &
-         long_name='Maximum carboxylation rate at 25 celcius for canopy layers', units='umol CO2/m**2/s', &
+         long_name='Maximum rate of electron transport at 25 Celcius for canopy layers', units='umol electrons/m**2/s', &
          interpinic_flag='interp', readvar=readvar, data=this%jmx25_z_patch)
-      call restartvar(ncid=ncid, flag=flag, varname='vcmx_prevyr', xtype=ncd_double,  &
+      call restartvar(ncid=ncid, flag=flag, varname='vcmx25_z_last_valid_patch:vcmx_prevyr', xtype=ncd_double,  &
```

Comment:
- [ ] Why do we need to keep `vcmx_prevyr` at all? Can this just be `vcmx25_z_last_valid_patch`

Reply:
This (and for jmx) is needed for backwards compatibility with the last iteration of restart files. New restart files will just use the new name.

Reply:
sound good, marked as resolved.

Reply:
Correct! That was for backward compatibility based on @billsacks suggestion.